README.md

```diff
@@ -63,4 +99,28 @@ class CustomCompletionFinder(argcomplete.CompletionFinder):
 completion_finder = cachedcomplete.cached_completion_finder(CustomCompletionFinder)()
 # Instead of argcomplete.autocomplete()
 completion_finder()
-```
\ No newline at end of file
+```
+
+### Specifying what files to track
+
+In order to specify which files to track changes in,
+add a comment with the wanted files and the prefix CACHEDCOMPLETE_HASH:
+
+```python
+# CACHEDCOMPLETE_HASH: file1.py
+```
+
+It also allows multiple files split to multiple comments, or within one comment.
+
+```python
+# CACHEDCOMPLETE_HASH: file1.py file2.json
+# CACHEDCOMPLETE_HASH: file3.py
+# CACHEDCOMPLETE_HASH: "file with spaces.txt"
+```
+
+And also allows passing up a directory to track all the files within
```

Comment:
Maybe add support for glob?

Reply:
No. It transforms the code from a simple, short line to a very complicated piece of code.

Reply:
OK. Your call.
If you want I can try to make it as least ugly as I can.

Reply:
@nivbromberg You're welcome to try, if it turns out well we'll merge it :)